### DISCUSSION TITLE
How to use with IntelliJ Dev containers?

### DISCUSSION THREAD

**Discussion by Samrose-Ahmed** [runfinch/finch#1014] [Q&A] [answered]
Whats the configuration to enter where it asks for Docker daemon, TCP socket, or SSH configuration?

**Comment by Shubhranshu153** [comment, marked as answer]
Thank you for your interest in finch.
Finch doesn't support api at the moment so unfortunately we wont be able to configure it.
Created an issue for tracking https://github.com/runfinch/finch/issues/1022
